FS.common.js

```diff
@@ -205,6 +206,13 @@ var RNFS = {
     return RNFSManager.copyFile(normalizeFilePath(filepath), normalizeFilePath(destPath), options).then(() => void 0);
   },
 
+  // Windows workaround for slow copying of large folders of files
+  copyFolder(filepath: string, destPath: string): Promise<void> {
+    if(isWindows) {
+      return RNFSManager.copyFolder(normalizeFilePath(filepath), normalizeFilePath(destPath));
+    }
```

Comment:
what happens on non-Windows? or does this need to somehow be marked as Windows-only ?

Reply:
This particular function needs to be marked as Windows-only. It exists as a workaround to make sure that copying directories with contents is done quickly. This can be done via JavaScript, but I suspect that the time it takes to make all the necessary calls increases the time needed to transfer all those files and contents from a fraction of a second to 30-40 seconds easily in a 60 megabyte folder with 60 different subfolders and files.

Is there a good (or at least, acceptable) way to do that?

Reply:
I'm not sure if there is a pattern for this but I want to make sure this doesn't break apps running on non-Windows devices

Reply:
Because it's only intended for Windows, would returning a promise rejection stating that this function is not available on other platforms be acceptable for other platforms?